.claude/skills/legal-petition/scripts/petition-to-pdf.py

```
@@ -0,0 +1,187 @@
+
+    try:
+        subprocess.run(
+            ["pandoc", "--version"],
+            capture_output=True,
+            check=True,
+        )
+    except (FileNotFoundError, subprocess.CalledProcessError):
+        missing.append("pandoc")
+
+    try:
+        import weasyprint  # noqa: F401
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)